<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>银行账户管理系统 刘洋 计科184</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>交易业务办理:</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>10</y>
       <width>441</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>交易类型:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="TransTypeComboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>交易金额:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TransAmountLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="SubmitTransButton">
        <property name="text">
         <string>提交</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>用户信息</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="UserNameLabel">
        <property name="text">
         <string>UserName</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>财产总量：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="UserBalanceLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>当前用户名下账户:</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>当前账户:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="AccountTypeComobox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>账户类型:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="AccountTypeRateTitleLabel">
        <property name="text">
         <string>储蓄利率:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>账户余额:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="AccountTypeLabel">
        <property name="text">
         <string>AccountTypeLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="AccountRateLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="AccountBalanceLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>上次操作:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="AccountOperateDateLabel">
        <property name="text">
         <string>2019-1-1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="AccountAnnualFeeTitleLabel">
        <property name="text">
         <string>年费:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="AccountCreditTitleLabek">
        <property name="text">
         <string>信用额度:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="AccountAnnualFeeLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="AccountCreditLabel">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDateEdit" name="TodayEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-M-d</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="nowDateLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>132</width>
      <height>19</height>
     </size>
    </property>
    <property name="text">
     <string>当前日期:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="LogTabWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>471</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="InstantLogTab">
     <attribute name="title">
      <string>实时日志</string>
     </attribute>
     <widget class="QTextBrowser" name="LogTextBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>451</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="LogInquireTab">
     <attribute name="title">
      <string>日志查询</string>
     </attribute>
     <widget class="QTextBrowser" name="LogInquireTextBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>441</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="LogButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>0</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="LogDateEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="displayFormat">
        <string>yyyy-M</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>选择日期：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="LogComboBox">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_U">
    <property name="title">
     <string>用户(&amp;U)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLogout"/>
    <addaction name="actionRegister_R"/>
   </widget>
   <widget class="QMenu" name="menu_Z">
    <property name="title">
     <string>账户(&amp;Z)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNewAccount"/>
   </widget>
   <addaction name="menu_U"/>
   <addaction name="menu_Z"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogout">
   <property name="text">
    <string>切换用户(&amp;T)</string>
   </property>
  </action>
  <action name="actionNewAccount">
   <property name="text">
    <string>创建帐户(&amp;C)</string>
   </property>
  </action>
  <action name="actionRegister_R">
   <property name="text">
    <string>注册用户(&amp;R)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
